the-content-guide/readme.md

```diff
@@ -7,14 +7,100 @@ The target audience are communication professionals. See it as a comprehensive g
+So how do you attract readers and keep them engaged? There is an increasing demand for *'high quality content'*. But what is high quality content? What ingrediënts make a nice meal that people will remember? Better yet, how do you get people to ask for and share the recipe?
 
-## User story:
+This guide will describe what ingredients work well together and how you can best present the meal. It will allow you to attract your target audience and keep them maximally engaged. The guide will also explain how you can test your business metrics and how to reach your audience. Most of all, how to create content you can be proud of!
 
-To know what makes good content — As a communication professional — I can read a comprehensive guide about content creation.
+## Originality & authenticity
+
+Good content is original. [Content Farms](https://en.wikipedia.org/wiki/Content_farm) are no longer wanted in these times. Search engines, such as Google, are getting better and better at detecting these 'clickbait sites' and these sites are not received well. Original content is sure to be shared more often. Even more so, why would you say something when it's not worth saying?
+
+> You share content because you want to achieve a certain goal. You want it to be read, and you want the reader to learn something from it.
+
+Apart from originality high quality content also makes a certain promise about the credibility of the text. When sources are been used they should provide authority. Good sources are crucial and make sure that a text is taken seriously.
+
+Be a guardian of the originality and authenticity of your content piece. It's easier sad than done while we all scrape information from different kind of sources. When it's authority in people you interview, articles, books or your own judgement, it's all based on other work. The difference where you as an author come in, is how you handle these sources. Combine them to share new insights.
+
+Repetition is not always a bad thing, it can be a powerful tool, but know that it will only work when it's used for original, compelling observations. High quality content is creative and has the potential to excite audiences.
+
+## Structure & readability
+
+Readers are looking for answers and want to find them in the easiest, fastest way. A well structured text gives guidance and leads the reader towards its conclusion in a gentle, organic fashion. A clear text consists of:
+
+* The title
+* An intro
+* Headers
+* **Bold** and *italic*  
+* Short paragraphs
+* Clear quotes
+
+A strong and remarkable *title* is important to attract your reader. A good title should activate, interest and excite. Often enough with newspaper articles—also the digital ones— the title is the key component in attracting the reader.
+
+If you have lured your reader with a fantastic title, he/she often decides whether to keep on reading or not based on the first few sentences. This brings us to the intro of the text and the importance hereof.
```

Comment:
"hereof" is a correct but very antiquated word. I've actually never heard it used in informal speech or writing before. Crazily, "thereof" is a more contemporary choice.